external-crates/move/crates/move-compiler/src/sui_mode/linters/share_owned.rs

```diff
@@ -95,11 +110,14 @@ impl SimpleAbsIntConstructor for ShareOwnedVerifier {
         {
             return None;
         }
-        Some(ShareOwnedVerifierAI)
+        if !calls_special_function(SHARE_FUNCTIONS, cfg) {
```

Comment:
Is this an optimization or it actually helps with accuracy?

Reply:
Optimization

Reply:
I wonder if it's worth an entirely separate visit and visitor modifications, but certainly a non-blocking issue for me